terraform/stack/provider.tf

```diff
@@ -1,5 +1,5 @@
 provider "aws" {
-  alias = "platform"
+  alias = "experience"
```

Comment:
Should this be in catalogue?

Reply:
What is "this"? The secret? The Terraform? Something else?

Reply:
the account, presumably?

Reply:
I don't think so?

I assume the service that will consume the items API will be part of the front-end, which runs in the experience account. Is there anything in the catalogue account that would be fetching this data?

Reply:
the frontend (in `experience`) consumes the works api, which is in `catalogue` - I guess we'd want to replicate that

Reply:
I think we're talking at cross-purposes, so let me be a bit more explicit:

Before this change, the `aws.platform` provider wasn't being used for anything. It's a holdover from some old Terraform.

The `aws.experience` provider is being used only to write an API key into Secrets Manager in the Experience account, so that a service running there can query the items API using API key authentication. The items API itself continues to run in the catalogue account.

Reply:
Ah, that makes more sense, sounds good :+1: